frontend_tests/node_tests/stream_data.js

```diff
@@ -454,6 +454,10 @@ zrequire('stream_data');
     stream_data.clear_subscriptions();
     stream_data.add_sub('India', india);
     assert(!stream_data.receives_desktop_notifications('India'));
-    global.blueslip.error = function() {};
+    global.blueslip.error = function () {};
     assert(!stream_data.receives_desktop_notifications('Indiana'));
+
+    assert(!stream_data.receives_audible_notifications('India'));
+    global.blueslip.error = function () {};
```

Comment:
Not sure why you need this?

Reply:
When I was running it on my computer, it seemed to crash without that line. I'm not sure why it's not behaving that way on your side. 

Should I just remove them and push again?

Reply:
Try deleting and running it again. Run the command `./tools/test-js-node` without the `--coverage` so we get a nice traceback. Tell me on chat.zulip.org if it still doesn't work.